src/app/content/highlights/reducer.ts

```diff
@@ -24,6 +25,10 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
     case getType(actions.createHighlight): {
       return {...state, highlights: [...state.highlights, action.payload]};
     }
+    case getType(actions.openMyHighlights):
+      return {...state, myHighlightsOpen: true};
+    case getType(actions.closeMyHighlights):
+      return {...state, myHighlightsOpen: false};
```

Comment:
should probably also set `myHighlightsOpen: false` on `locationChange` up on line 23